### PR TITLE
add ignore option

### PR DESCRIPTION
**Pull request by janpaepke**
As discussed here:
https://github.com/mmkal/slonik-tools/issues/385

- prevent all queries, missing SELECT, RETURNING or VALUES, from even being considered 
- do not process queries, which have a `typegen-ignore` comment in them
- reflect in logs, when explicitly ignoring
- update docs

### CODE REVIEW

File: packages/typegen/src/util.ts [hunk 1]
```diff
@@ -78,3 +82,33 @@ export const changedFiles = (params: {since: string; cwd: string}) =>
...
+  const isSqlIdentifier = (node: ts.Node) => ts.isIdentifier(node) && node.getText() === 'sql'
+  const isSqlPropertyAccessor = (e: ts.Expression) => ts.isPropertyAccessExpression(e) && isSqlIdentifier(e.name)
+  /**
+   * Checks if the supplied node is an sql template literal
+   */
+  const isSqlLiteral = (node: ts.Node): node is ts.TaggedTemplateExpression =>
+    ts.isTaggedTemplateExpression(node) && (isSqlIdentifier(node.tag) || isSqlPropertyAccessor(node.tag))
+  return {
+    isSqlLiteral,
+  } as const
```
Comment (mmkal) [inline packages/typegen/src/util.ts:96]: minor: what's the advantage of `as const` here?

**Review by mmkal — APPROVED** [review]
Only one tiny comment - I'm happy for you to merge with or without addressing. This is awesome!

File: packages/typegen/src/util.ts (hunk 1 above)
Comment (mmkal) [reply packages/typegen/src/util.ts:96]: Ok, sure. I generally don’t use readonly modifiers - I just don’t mutate objects but no harm in this.

**Comment by janpaepke** [comment]
Good to merge then?

**Comment by mmkal** [comment]
Yep sorry was leaving you to do it, will merge and cut a release now

**Pull request merged**